src/frontend/src/storage/index.ts

```diff
@@ -135,6 +138,27 @@ const mostUnused = (anchors: Anchors): string | undefined => {
 /** Best effort to migrate potentially old localStorage data to
  * the latest format */
 const migrated = (): Anchors | undefined => {
+  // Try to read the "v1" (localStorage.anchors) storage and return that if found
+  const v1 = migratedV1();
```

Comment:
out of curiosity, why calling it v1 and v0 instead of using the names anchors and userNumber?

Reply:
Because I'm expecting more migrations in the future and don't want to give them all names :)